terraform/modules/r53/main.tf

```diff
@@ -0,0 +1,283 @@
+resource "aws_route53_zone" "hashbang" {
+    name = "hashbang.sh"
+}
+
+// A
+resource "aws_route53_record" "mail-ipv4" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "mail.${aws_route53_zone.hashbang.name}"
+    type = "A"
+    ttl  = "1800"
+    records = ["104.236.46.93"]
+}
+
+resource "aws_route53_record" "sfo1-irc-ipv4" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "sfo1.irc.${aws_route53_zone.hashbang.name}"
+    type = "A"
+    ttl  = "1800"
+    records = ["104.236.128.136"]
+}
+
+resource "aws_route53_record" "lon1-irc-ipv4" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "lon1.irc.${aws_route53_zone.hashbang.name}"
+    type = "A"
+    ttl  = "1800"
+    records = ["178.62.33.133"]
+}
+
+// GEO
+resource "aws_route53_record" "irc-ipv4-us" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "irc.${aws_route53_zone.hashbang.name}"
+    type = "A"
+    ttl  = "1800"
+    records = ["104.236.128.136"]
```

Comment:
Will that result in users being only pointed to `sfo1.irc.hashbang.sh` even when it is down?
Can we make it use a status check?